mobile/build.gradle.kts

```diff
@@ -67,6 +67,7 @@ dependencies {
   androidTestImplementation(libs.androidx.test.truth)
   androidTestImplementation(libs.androidx.test.espresso)
   androidTestImplementation(libs.compose.ui.test.junit4)
+  androidTestImplementation(libs.kotlin.reflect)
```

Comment:
Comment: I do not really understand why we need to add the library reflect to the tests implementation. Do we run the tests for the library or do we provide this library for our tests?

Reply:
The library is an interface which defines how to get data from Firestore.

An implementation which uses the actual Firestore SDK is provided in the application, and is tested with standard mocking. Also, a mock implementation is provided in the androidTest source set, which stores data in memory and makes use of reflection.